app/assets/stylesheets/dfe/side-nav.scss

```diff
@@ -234,4 +234,10 @@ label.menu-icon svg {
   .content-with-side-menu {
     margin-top: 0px;
   }
-}
\ No newline at end of file
+}
+
+.govuk-list--bullet {
```

Comment:
This declaration will restyle all uses of this class not just in the side nav.

Reply:
This bug has now been resolved by removing this styling and updating side-nav to use `.govuk-grid-column-three-quarters`